### ISSUE TITLE
Xyz serial number... For those already switched over

### ISSUE THREAD

**Issue by jack30977** [repetier/Repetier-Firmware#933]
So since you guys converted your 3d printers to repetier and I am a dumbnut who can't figure this one out. I converted back to xyz and I am asking if I could have anyone's serial number so that way I can use it to register my own because I picked mine up from a dumpster. Therefor I do not have access to the back panel.
Thanks if you can give me your serial number,

**Comment by MasterCATZ** [comment]
do you still need serial ? I lent my Da Vinci 2.0 Duo out yesterday but can ask them to get the number for me , tho I do have its box here still that might have it

**Comment by jack30977** [comment]
Yeah I still need a serial number. Its located on the back cover. Thank you for responding back to me
